fdbserver/workloads/GetMappedRange.actor.cpp

```diff
@@ -433,13 +486,16 @@ struct GetMappedRangeWorkload : ApiWorkload {
 		} else if (r < 0.75) {
 			matchIndex = MATCH_INDEX_UNMATCHED_ONLY;
 		}
+		(const_cast<ServerKnobs*> SERVER_KNOBS)->STRICTLY_ENFORCE_BYTE_LIMIT = deterministicRandom()->coinflip();
 		wait(self->scanMappedRange(cx, 10, 490, mapper, self, matchIndex));
 
 		{
 			Key mapper = getMapper(self, true);
 			wait(self->scanMappedRange(cx, 10, 490, mapper, self, MATCH_INDEX_UNMATCHED_ONLY, true));
 		}
 
+		// reset it to default
+		(const_cast<ServerKnobs*> SERVER_KNOBS)->STRICTLY_ENFORCE_BYTE_LIMIT = false;
```

Comment:
It's better to save the default value and restore from that, because the default may not be `false` (e.g., after a change) and then this is wrong.

Reply:
sounds good, will have another PR for this change